src/components/app.st.css

```diff
@@ -14,11 +15,10 @@
 }
 
 :vars {
-    bgMain: rgb(42, 42, 42);
-    bgContrast: rgba(255, 255, 255, 0.9);
+    dark: stArray(#252526, #1e1e1e);
```

Comment:
maybe `stMap` would allow me to understand what is the semantics of these colors or a hint in the name would have been helpful like maybe it's `darkShades` or maybe it `darkPalette`